static/js/place/chart.tsx

```diff
@@ -70,6 +77,14 @@ interface ChartPropType {
    * Scale number
    */
   scaling?: number;
+  /**
+   * Geojson data for places one level down of current dcid.
+   */
+  geoJsonData?: unknown;
```

Comment:
would be nice to use the actual type for geoJson

Reply:
we can also add a type for it first (which is set to unknown for now) with a todo to add the complete type later